db/re/item_db.conf

```diff
@@ -5855,9 +5855,11 @@ item_db: (
 	View: 3
 	Script: <"
 		bonus bFlee2,10;
-		if(JobLevel>=70) autobonus "{ bonus bBaseAtk,50; }",10,10000,BF_WEAPON,"{ specialeffect2 EF_POTION_BERSERK; }";
+		if(JobLevel=70)||(JobLevel>=50)&&(Class==Job_Rune_Knight||(JobLevel>=50)&&Class==Job_Rune_Knight_T) {
```

Comment:
using JobLevel=70 instead of JobLevel>=70 will break this item for all servers that uses a higher max jlvl, and by the way that statement doesn't looks logic since it will always be true for both cases

jlvl >= 50 will trigger the jlvl =70 as well

I'd use something like:
if(JobLevel >= 70) && Class == Job_Rune_Knight_T) || (JobLevel >= 50 && Class == Job_Rune_Knight)

instead :+1: